ui.config/src/main/content/jcr_root/apps/groovyconsole/osgiconfig/config/org.apache.sling.jcr.repoinit.RepositoryInitializer~aemgroovyconsole.config

```diff
@@ -0,0 +1,9 @@
+scripts=["
+    create path /var/groovyconsole/audit(nt:folder)
```

Comment:
Using `nt:folder` here will not allow to create audit nodes, `nt:unstructured` should work though. 

`create path /var/groovyconsole(sling:Folder)/audit(nt:unstructured)`

Reply:
Will this be always executed before other OSGi services are activated ? I see that some of the services requires the user and permissions to activate (see  `DefaultAuditService` activate method) and if this will be executed later then activate will throw an exception and service will not start causing chain of inactive services.

Reply:
It will happen before services (transitively) referencing the SlingRepository service will start (look at https://sling.apache.org/documentation/bundles/repository-initialization.html)